internal/testplanet/node.go

```diff
@@ -175,12 +175,11 @@ func (node *Node) initOverlay(planet *Planet) error {
 	if err != nil {
 		return utils.CombineErrors(err, routing.Close())
 	}
-	node.Kademlia = kad
 
+	node.Kademlia = kad
 	node.StatDB = node.Database.StatDB()
-
 	node.Overlay = overlay.NewCache(teststore.New(), node.StatDB)
-	node.Discovery = discovery.NewDiscovery(node.Overlay, node.Kademlia, node.StatDB)
+	node.Discovery = discovery.NewDiscovery(zap.L(), node.Overlay, node.Kademlia, node.StatDB)
```

Comment:
zap.L() -> node.Log.
Rule of thumb is that only the configs.go files should reference the global logger

Reply:
i second that

Reply:
`node.Log.Named("discovery")`